megamek/src/megamek/common/Entity.java

```diff
@@ -9313,12 +9313,18 @@ public String toString() {
      */
     public String statusToString() {
```

Comment:
Related to this PR, but something you inherited and not required to be addressed here: these accessibility status string builders should be hoisted out to help us avoid the bloat we have going on inside of `Entity`.

Reply:
Hmm good point, I wanted to keep it in the classes so that if updates happened the accessibility wouldn't be forgotten. But lifting it out might work better. I will keep it in mind for future updates, I will probably need to do a pass over most of these strings anyway.